symphony/bdk/core/auth/bot_authenticator.py

```diff
@@ -1,12 +1,13 @@
 from abc import ABC, abstractmethod
-from symphony.bdk.gen.login_api.authentication_api import AuthenticationApi
-from symphony.bdk.gen.login_model.authenticate_request import AuthenticateRequest
-from symphony.bdk.gen.exceptions import ApiException
-from symphony.bdk.gen.api_client import ApiClient
-from symphony.bdk.core.auth.jwt_helper import create_signed_jwt
+
 from symphony.bdk.core.auth.auth_session import AuthSession
 from symphony.bdk.core.auth.exception import AuthUnauthorizedException
+from symphony.bdk.core.auth.jwt_helper import create_signed_jwt
```

Comment:
in create_signed_jwt we still have this magic 5*58, could we make it a constant and document that JWT needs to have a ttl of 5 minutes max